Assets/Scripts/ArenaManager.cs

```diff
@@ -129,10 +124,30 @@ public void ReturnMoveMarker(Interactable moveMarker)
         
         private void SetTeamColors()
         {
-            foreach (var team in teams)
+            for (int i = 0; i < teams.Length; i++)
```

Comment:
I would actually the old MAX_TEAMS to 8 or something because what this used to do was take samples on the color wheel of half MAX_TEAMS and then would reduce saturation and shift the colors on the color wheel a little so you get more pastel versions with slight hue changes.  This helps visually distinguish them.  I think you can just use full saturation ones for now and if we add up to 8 we can start leveraging the lower saturation colors.